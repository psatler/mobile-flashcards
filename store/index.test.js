import store from './'

it('store should match snapshot', () => {
    const state = store.getState();
    expect(state).toMatchSnapshot();
});