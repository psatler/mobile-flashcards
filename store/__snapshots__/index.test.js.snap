// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store should match snapshot 1`] = `
Object {
  "deckReducer": Object {},
}
`;
